torch/csrc/jit/interpreter.h

```diff
@@ -2,6 +2,9 @@
 #include <memory>
 #include <vector>
 
+#include "torch/csrc/jit/generated/aten_dispatch.h"
```

Comment:
Let's move these includes to `interpreter.cpp`, they are not needed in the header and slow down compilation.